src/github/pullRequestManager.ts

```diff
@@ -753,14 +753,18 @@ export class PullRequestManager {
 	async deleteReview(pullRequest: PullRequestModel): Promise<{ deletedReviewId: number, deletedReviewComments: Comment[] }> {
 		const pendingReviewId = await this.getPendingReviewId(pullRequest);
 		const { mutate } = await pullRequest.githubRepository.ensure();
-		const { data } = await mutate<DeleteReviewResponse>({
+		const { data, errors } = await mutate<DeleteReviewResponse>({
 			mutation: queries.DeleteReview,
 			variables: {
 				input: { pullRequestReviewId: pendingReviewId }
 			}
 		});
 
-		const { comments, databaseId } = data.deletePullRequestReview.pullRequestReview;
+		if (errors) {
+			throw new Error(errors.map(err => err.message).join(', '));

```

Comment:
`err` leads a noImplicitAny error.

Reply:
huh, I'm not seeing that when I build

Reply:
My ts server didn't pick the typings, reloading the project fixes the problem. Sorry for the false alarm.